lte/gateway/python/magma/enodebd/state_machines/enb_acs_impl.py

```diff
@@ -113,13 +113,15 @@ def transition(self, next_state: str) -> Any:
             'State transition from <%s> to <%s>',
             self.state.__class__.__name__, next_state,
         )
-        self.state.exit()
-        self.state = self.state_map[next_state]
-        self.state.enter()
+        if self.state:
```

Comment:
Looking at the code I wonder how the state can ever actually be `None`. It's initialized by the `_start_state_machine` call in the constructor. Maybe you can also solve the type error by not setting it to `None` initially and just giving a type annotation instead, i.e. setting `self.state: EnodebAcsState` in `__init__`.